Dockerfile

```diff
@@ -129,6 +147,8 @@ COPY --from=centos7-build /google-cloud-ops-agent*.rpm /
 
 FROM rockylinux:8 AS centos8-build-base
 
+COPY --from=cmake-install /cmake.sh /cmake.sh
```

Comment:
Only needed if you actually install `cmake`, right? You could pull `cmake` installation out into a couple of separate layers — that way you could add both:
```
COPY --from=cmake-install /cmake.sh /cmake.sh
RUN set -x; bash /cmake.sh --skip-license --prefix=/usr/local
```
at once when needed…

Reply:
I'm not sure I follow; are you suggesting that we leave only the following in the header:

```
# Manually prepare a new enough version of CMake.
# This should be used on platforms where the default package manager
# does not provide a new enough version (we require >= 3.12).
FROM alpine:latest AS cmake-amd64

ENV hash=4d98de8d605da676e71a889dd94f80c76abb377fade2f21e3510e62ece1e1ada
ADD https://github.com/Kitware/CMake/releases/download/v3.25.2/cmake-3.25.2-linux-x86_64.sh \
    /cmake.sh

FROM alpine:latest AS cmake-arm64

ENV hash=73a35cab2174a3eb8f35083d55c80871185dc3808f3dae3558cd5fbdb29a4614
ADD https://github.com/Kitware/CMake/releases/download/v3.25.2/cmake-3.25.2-linux-aarch64.sh \
    /cmake.sh
```

and then move the `COPY / RUN` lines to each image on an as-needed basis?

Reply:
Thanks, makes sense -- updated.